src/styles/main.scss

```
@@ -1,3 +1,48 @@
-body {
-  margin: 0;
+@import "./colors.scss";
+@import "./header.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
+@mixin columns($column-count) {
+  grid-template-columns: repeat($column-count, 200px);
+}
+
+.page {
+
+  &__body {
+    margin: 0;
+    font-family: "roboto", sans-serif;
+  }
+
+  &__header {
+    padding: 0 50px;
+    box-sizing: border-box;
+  }
+
+  &__cards {
+    padding: 50px 40px;
+    box-sizing: border-box;
+
+    display: grid;
+    @include columns(1);
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      & {
+        @include columns(2);
+      }
+    }
```

Let's fix it everywhere
```suggestion
    @media (min-width: 488px) {
      @include columns(2);
    }
```